# ChEMBL-NTD

Welcome to the **ChEMBL - Neglected Tropical Disease** archive, a repository for Open Access primary screening and medicinal chemistry data directed at neglected diseases - endemic tropical diseases of the developing regions of the Africa, Asia, and the Americas. The primary purpose of ChEMBL-NTD is to provide a freely accessible and permanent archive and distribution centre for deposited data. ChEMBL-NTD is a subset of the data in the free medicinal chemistry and drug discovery database [ChEMBL](http://www.ebi.ac.uk/chembl/beta/). We actively encourage download and use of the [data](downloads/).

ChEMBL-NTD is maintained by EMBL-EBI at Hinxton in the United Kingdom, and has received no funding from any commercial organisation, individual or investor. More information on the EMBL-EBI can be found [here](http://www.ebi.ac.uk/).

The deposited data are hosted on the servers of the EMBL-EBI at Hinxton in the United Kingdom. The static pages are hosted by [GitBook](https://www.gitbook.com/).

If you have questions about the data - [contact us!](mailto:chembl-help@ebi.ac.uk?subject=\[ChEMBL-NTD]%20Data%20query)

If you wish to deposit data - [contact us!](mailto:chembl-help@ebi.ac.uk?subject=\[ChEMBL-NTD]%20Contribute%20data%20query%20link)

## ChEMBL-NTD Terms of Use

We encourage all users to download, copy and redistribute these data as needed. However, in the spirit of open collaboration and to enable rapid development of new therapeutics for neglected disease, we encourage following these basic principles:

* Users who annotate, add to, or modify these data in a way that adds significant value are encouraged to release their work to the public domain, ideally by re-contributing their findings to ChEMBL-NTD.
* When these data are used or cited in a paper or other scholarly work please reference the citation provided in each deposition set.

Access to the ChEMBL-NTD data is under the [EMBL-EBI's standard Terms of Use](http://www.ebi.ac.uk/Information/termsofuse.html)

##

## Deposited Set 27: 23 Jan 2024 - Identification of inhibitors of _Trypanosoma cruzi_ M17 leucyl-aminopeptidase (TcLAP/LAPTc)&#x20;

A library of 3328 compounds that harbour common protease inhibitor motifs was screened at 30 µM against LAPTc using a RapidFire-MS method.

If you wish to use the data please cite: Maikel Izquierdo, De Lin, Sandra O’Neill, Lauren A. Webster, Christy Paterson, John Thomas, Mirtha Elisa Aguado, Enrique Colina Araújo, Daniel Alpízar-Pedraza, Halimatu Joji, Lorna MacLean, Anthony Hope, David W. Gray, Martin Zoltner, Mark C. Field, Jorge González-Bacerio and Manu De Rycker. Identification of a potent and selective LAPTc inhibitor by RapidFire-Mass Spectrometry, with antichagasic activity. PLOS NTD. 2024 (Accepted).

## Deposited Set 26: 3 March 2023 – Using ChEMBL to complement schistosome drug discovery

A library of 80 compounds were tested _in vitro_ on different life cycle stages of the parasite _Schistosoma mansoni_.

·       _In vitro_ activity on schistosomula (larva stage of the parasite)

Mechanically transformed schistosomula were automatically dispensed into a 384-well plate (120 parasite/well). The parasites were dosed with compounds and incubated in a humidified environment at 5% CO2 and 37°C for 72 h. Following incubation, com-pound-induced effects were assessed using an in-house facility, Roboworm, which quantifies both larva motility and phenotype. Preliminary compound screens were performed at a single-point concentration of 10 and 50 µM. Three independent screens were performed including two technical duplicates for each compound/concentration. Each screen contained the positive and negative controls (Auranofin - AUR at 10 μM final concentration in 0.625% DMSO, and 0.625% DMSO, respectively). The phenotype and motility scores were used to evaluate whether a compound displayed anti-schistosomula activity; here, -0.15 and -0.35 defined threshold anti-schistosomula values for phenotype and motility scores respectively. Secondary dose-response titrations were performed for all compounds identified as hits at 10 µM. At least two titrations were performed for each compound. The concentration range included 0.313, 0.625, 1.250, 2.500, 5 and 10 µM (with each concentration point in duplicate). Experimental data (i.e., phenotype and motility scores) were used to compute EC50 values using GraphPad Prism 7.02.

&#x20;

·       _In vitro_ activity on adult worms

Adult worms (1 worm pair/1 ml of adult worm media) were dosed with 10 µM of each compound (in 0.1% DMSO). Negative (0.1% DMSO) and positive control (praziquantel - 10 µM in 0.5% DMSO) treatments were included in each replicate (at least three independent experiments). Compound and parasite co-cultures were incubated for 72 h in a humidified environment at 5% CO2, 37°C. Parasite motility after compound treatment was assessed at three time points (24, 48 and 72 h) by a digital image processing-based system (WormassayGP). The final readout related to the 72 h time point is defined as worm movement inhibition (compared to the DMSO control).

## Deposited Set 25: 1st July 2021 – The discovery of a novel series of compounds with single-dose efficacy against juvenile and adult Schistosoma species

Treatment and control of schistosomiasis depends on a single drug, praziquantel, but this is not ideal for several reasons including lack of potency against the juvenile stage of the parasite, dose size, and risk of resistance. We have optimised the properties of a series of compounds we discovered through high throughput screening and have designed candidates for clinical development. The best compounds demonstrate clearance of both juvenile and adult S. mansoni worms in a mouse model of infection from a single oral dose of < 10 mg/kg. Several compounds in the series are predicted to treat schistosomiasis in humans across a range of species with a single oral dose of less than 5 mg/kg

If you wish to use the data, please cite: J. Mark F. Gardner, Nuha R. Mansour, Andrew S. Bell, Helena Helmby, Quentin Bickle. The discovery of a novel series of compounds with single-dose efficacy against juvenile and adult Schistosoma species. PLOS NTD. 2021 (Accepted).

## Deposited Set 24:&#x20;

_\[Placeholder entry for Fraunhofer Institute SARS-CoV-2 data that needed an FTP server to host it]_

## Deposited Set 23: 28th January 2020 – Probing the Open Global Health Chemical Diversity Library for multistage-active starting points for next-generation antimalarials

Most phenotypic screens aiming to discover new antimalarial chemotypes begin with low cost, high-throughput tests against the asexual blood stage (ABS) of the malaria parasite lifecycle. Compounds active against the ABS are then sequentially tested in more difficult assays that predict whether a compound has other beneficial attributes. Although applying this strategy to new chemical libraries may yield new leads, repeated iterations may lead to diminishing returns and the rediscovery of chemotypes hitting well-known targets. Here we adopted a different strategy to find starting points, testing \~70,000 open source small molecules from the Global Health Chemical Diversity Library for activity against the liver stage, mature sexual stage and asexual blood stage malaria parasites in parallel. In addition, instead of using an asexual assay that measures accumulated parasite DNA in the presence of compound (SYBR green), a real time luciferase-dependent parasite viability assay was used that distinguishes slow acting (delayed death) from fast-acting compounds. Among 382 scaffolds with activity confirmed in dose response (<10µM), we discovered 68 novel delayed-death, 84 liver-stage, and 68 stage V gametocyte inhibitors as well. Although 89% of evaluated compounds had activity in only a single lifecycle stage, we discovered six potent (half-maximal inhibitory concentration <1µM) multistage scaffolds, including a novel cytochrome bc1 chemotype. Our data further show the luciferase-based assays have higher sensitivity. Chemoinformatic analysis of positive and negative compounds identified scaffold families with a strong enrichment for activity against specific or multiple stages.

[UCSD](https://ucsd.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you wish to use the data please cite: Abraham M, Gagaring K, Martino ML, Vanaerschot M, Plouffe DM, Calla J, Godinez-Macias KP, Du AY, Wree M, Antonova-Koch Y, Eribez K, Luth MR, Ottilie S, Fidock DA, McNamara CW, Winzeler EA. Probing the Open Global Health Chemical Diversity Library for multistage-active starting points for next-generation antimalarials. ACS Infectious Diseases. 2020 (Accepted).

## Deposited Set 22: 20th December 2018 – UCSD Winzeler Lab liver-stage Plasmodium screening

The dataset lists compounds that may protect against malaria based on their ability to inhibit luciferase-expressing Plasmodium berghei from developing in hepatoma cells. The dataset is the result of screening a 500,000 compound small molecule library in this assay. Subsets of the compounds were further tested in dose response format for toxicity, biochemical luciferase inhibition, against P falciparum asexual blood stages, P. vivax hepatic stages and for inhibition of P. falciparum mitochondrial function. The dataset is the result of collaboration between the [Winzeler lab at the University of California, San Diego](http://winzeler.ucsd.edu/) and the [Medicines for Malaria Venture](https://www.mmv.org/) with assistance from many others, including members of the [Malaria Drug Accelerator](http://winzeler.ucsd.edu/malda/) and the Genomics Institute of the Novartis Research Foundation.

[UCSD](https://ucsd.edu/) and [MMV](https://www.mmv.org/) do not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you wish to use the data please cite: Antonova-Koch Y, Meister S, Abraham M, Luth MR, Ottilie S, Lukens AK, Sakata-Kato T, Vanaerschot M, Owen E, Jado Rodriguez JC, Maher SP, Calla J, Plouffe D, Zhong Y, Chen K, Chaumeau V, Conway AJ, McNamara CW, Ibanez M, Gagaring K, Serrano FN, Eribez K, Taggard CM, Cheung AL, Lincoln C, Ambachew B, Rouillier M, Siegel D, Nosten F, Kyle DE, Gamo FJ, Zhou Y, Llinás M, Fidock DA, Wirth DF, Burrows J, Campo B, Winzeler EA. Open-source discovery of chemical leads for next-generation chemoprotective antimalarials. Science. 2018 Dec 7;362(6419). pii: eaat9446. doi: [10.1126/science.aat9446](http://science.sciencemag.org/content/362/6419/eaat9446).

## Deposited Set 21: 14th October 2016 – MMV Pathogen Box

Welcome to first [Medicines for Malaria Venture](http://www.mmv.org/) Pathogen Box dataset. The Pathogen Box contains 400 diverse, drug-like molecules active against neglected diseases of interest. This dataset contains biological activity confirmation data performed in the following disease areas: Tuberculosis, Malaria, Chagas disease, Leishmaniasis, Human African Trypanosomiasis, Cryptosporidiosis, Lymphatic Filariasis, Onchocerciasis, Schistosomiasis, Dengue, Chikungunya, Toxoplasmosis.

[Medicines for Malaria Venture](http://www.mmv.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the data please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd). List of contributors:

* Nathan group, Weill Cornell Medical College
* Barry Lab, NIAID NIH
* Winzeler lab, UCSD
* Laboratory of Microbiology, Parasitology and Hygiene (LMPH), Univ. of Antwerp
* London School of Hygiene & Tropical Medicine
* Parasite Chemotherapy Unit at Swiss Tropical and Public Health Institute
* Huston group at the University of Vermont
* Castellanos group at UTMB, Galveston, Texas
* Sakanari Lab, Univ. of Calif. San Francisco
* Townson (Simon) Lab, Imperial College London
* Fidelis Cho-Ngwa Lab, University of Buea, Buea, Cameroon
* Laboratory of Molecular Parasitology at the New York Blood Center
* Bickle Lab, London School of Hygiene & Tropical Medicine
* Keiser Lab, Swiss Tropical and Public Health Institute
* Caffrey group at the Center for Discovery and Innovation in Parasitic Diseases, Skaggs School of Pharmacy and Pharmaceutical Sciences, UC San Diego
* University of Cape Town
* Shanmugam group at CSIR-National Chemical Laboratory, Pune (India)
* Novartis Institute for Tropical Disease-Singapore
* Abbvie

## Deposited Set 20: 2nd August 2016 - SwissTPH T. brucei screening hits

Welcome to the [SwissTPH](http://www.swisstph.ch/) T. brucei screening hits dataset. The protozoan parasites Trypanosoma brucei spp., the causative agents of human sleeping sickness, are transmitted by tsetse flies. During differentiation from bloodstream forms to procyclic forms, which naturally occur in the midgut of the fly, the trypanosomes shed their coat of variant surface glycoprotein (VSG). This process would be lethal for the parasites while in the bloodstream of an infected patient. We searched in a medium-throughput screen for triggers that artificially stimulate differentiation towards procyclic forms at 37°C and identified 28 hits. The differentiation-inducing hits could facilitate studies on the regulation of differentiation and serve as lead scaffolds for medicinal chemistry in order to develop new treatments for sleeping sickness.

[SwissTPH](http://www.swisstph.ch/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the data please include the link to [ChEMBL-NTD](https://www.ebi.ac.uk/chemblntd) and adapt the following citation language: Wenzler T, Schumann Burkard G, Schmidt RS, Mäser P, Bergner A, Roditi I, Brun R. A new approach to chemotherapy: drug-induced differentiation kills African trypanosomes. Sci Rep. 2016 Mar 2;6:22451. [doi:10.1038/srep22451](http://www.nature.com/articles/srep22451).

## Deposited Set 19: 5th March 2016 - UW kinase screening hits

[UW](http://www.washington.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the UW kinase screening dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Gregory J. Crowther, Heidi K. Hillesland, Katelyn R. Ceylon, Molly C. Reid, Maria Jose Lafuente-Monasterio, Sonja Ghidelli-Disse, Stephen E. Leonard, Panqing He, Jackson C. Jones, Mallory M. Krahn, Jack S. Mo, Kartheek S. Dasari, Anna M. W. Fox, Markus Boesche, Majida El Bakkouri, Kasey L. Rivas, Didier Leroy, Raymond Hui, Gerard Drewes, Dustin J. Maly, Wesley C. Van Voorhis, Kayode K. Ojo. Biochemical Screening of Five Protein Kinases from Plasmodium falciparumagainst 14,000 Cell-Active Compounds. PLoS ONE, 2016. [doi:10.1371/journal.pone.0149996](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0149996).

## Deposited Set 18: 15th February 2016 - UCSD Winzeler Lab (hits from exoerytrocytic-stage screening)

Welcome to the [UC San Diego School of Medicine Winzeler Lab](http://winzeler.ucsd.edu/) exoerytrocytic-stage screening hits dataset. We have developed a luciferase-based phenotypic screen of malaria exoerythrocytic-stage parasites optimized for a 1536-well format. This assay uses the exoerythrocytic-stage of the rodent malaria parasite, Plasmodium berghei, and a human hepatoma cell line. We use this assay to evaluate several biased and unbiased compound libraries, including two small sets of molecules (400 and 89 compounds, respectively) with known activity against malaria erythrocytic-stage parasites and a set of 9,886 Diversity-Oriented Synthesis (DOS)-derived compounds. Of the compounds screened we obtain hit rates of 12-13% and 0.6% in preselected and naïve libraries, respectively, and identify 52 compounds with exoerythrocytic-stage activity less than 1mM, and having minimal host cell toxicity. Our data demonstrate the ability of this method to identify compounds known to have causal prophylactic activity in both human and animal models of malaria, as well as novel compounds, including several exclusively active against parasite exoerythrocytic stages.

[UCSD](https://ucsd.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the UCSD exoerythrocytic-stage dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Justine Swann, Victoria Corey, Christina A. Scherer, Nobutaka Kato, Eamon Comer, Micah Maetani, Yevgeniya Antonova-Koch, Christin Reimer, Kerstin Gagaring, Maureen Ibanez, David Plouffe, Anne-Marie Zeeman, Clemens H. M. Kocken, Case W. McNamara, Stuart L. Schreiber, Brice Campo, Elizabeth A. Winzeler, Stephan Meister. A high-throughput luciferase-based assay for the discovery of therapeutics that prevent malaria. ACS Infectious Diseases, 2016. [doi:10.1021/acsinfecdis.5b00143](http://pubs.acs.org/doi/abs/10.1021/acsinfecdis.5b00143).

## Deposited Set 17: 25th January 2016 - 2nd Tres Cantos TB TCAMS dataset (hits from M. Tuberculosis phenotypic screening)

[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the second GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Rebollo-Lopez, M. J.; Lelièvre, J.; Alvarez-Gomez, D.; Castro-Pichel, J.; Martínez-Jiménez, F.; Papadatos, G.; Kumar, V.; Colmenarejo, G.; Mugumbate, G.; Hurle, M.; Barroso, V.; Young, R. J.; Martinez-Hoyos, M.; González del Río, R.; Bates, R. H.; Lopez-Roman, E. M.; Mendoza-Losana, A.; Brown, J. R.; Alvarez-Ruiz, E.; Marti-Renom, M. A.; Overington, J. P.; Cammack, N.; Ballell, L.; Barros-Aguire, D., Release of 50 new, drug-like compounds and their computational target predictions for open source anti-tubercular drug discovery. PLoS ONE 2015, 10, e0142293. \[[doi:10.1371/journal.pone.0142293 ](http://dx.doi.org/10.1371/journal.pone.0142293)]

## Deposited Set 16: 25th November 2015 - TropIQ SMFA Dataset

[TropIQ](http://tropiq.nl/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 15: 18th August 2015 - DNDi Dataset: Antiprotozoal activity profiling of approved drugs

[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the DNDi drug repositioning dataset please include the link to ChEMBL-NTD and adapt the following citation language: Kaiser M, Mäser P, Tadoori LP, Ioset JR, Brun R. Antiprotozoal Activity Profiling of Approved Drugs: A Starting Point toward Drug Repositioning. PLoS One. 2015 10(8): e0135556. \[[doi:10.1371/journal.pone.0135556](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0135556)]

## Deposited Set 14: 5th March 2015 - GSK TCAKS Dataset (hits from Leishmania donovani, Trypanosoma cruzi and Trypanosoma brucei brucei phenotypic screening)

[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to ChEMBL-NTD.

If you publish on, or wish to reference the GSK TCAKS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Imanol Pena, M. Pilar Manzano, Juan Cantizani, Albane Kessler, Julio Alonso-Padilla, Ana I. Bardera, Emilio Alvarez, Gonzalo Colmenarejo, Ignacio Cotillo, Irene Roquero, Francisco de Dios-Anton, Vanessa Barroso, Ana Rodriguez, David W. Gray, Miguel Navarro, Vinod Kumar, Alexander Sherstnev, David Drewry, James R. Brown, Jose M. Fiandor & J. Julio Martin. (2015), New Compound Sets Identified from High Throughput Phenotypic Screening against Three Kinetoplastid Parasites: An Open Resource. \[doi:[10.1038/srep08771](http://dx.doi.org/10.1038/srep08771)]

## Deposited Set 13: 4th February 2015 - St. Jude Children's Research Hospital Dataset

The [St. Jude Children's Research Hospital](http://www.stjude.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 12: 11th October 2014 - Antimalarial screening hits dataset

[MMV](http://www.mmv.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the MMV dataset set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Avery, V. M., Bashyam, S., Burrows, J. N., Duffy, S., Papadatos, G., Puthukkuti, S., et al. (2014). Screening and hit evaluation of a chemical library against blood-stage Plasmodium falciparum. Malaria Journal, 13, 190. \[[doi:10.1186/1475-2875-13-190](http://dx.doi.org/10.1186/1475-2875-13-190)]

## Deposited Set 11: 12th November 2013 - UCSF DeRisi Lab MMV Dataset

This work was funded by a [Grand Challenges Exploration grant from the ](http://www.grandchallenges.org/Explorations/Pages/Introduction.aspx)[Bill and Melinda Gates Foundation](http://www.gatesfoundation.org/) and [The Howard Hughes Medical Institute](http://www.hhmi.org/).

UC San Francisco and the Howard Hughes Medical Institute do not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 10: 28th October 2013 - Selection and optimization of hits from a high-throughput phenotypic screen against Trypanosoma cruzi

**15th April 2014** - Two additional T. cruzi datasets have been deposited by DNDi. The two new fenarimol series datasets, from which preclinical candidates EPL-BS0967 and EPL-BS1246 were identified, are available to download [here](downloads/#deposited-set-8-dndi-t-b-brucei-dataset-29-th-november-2012).

[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the DNDi T. cruzi dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Martine Keenan, Paul W. Alexander, Jason H Chaplin, Michael J Abbott, Hugo Diao, Zhisen Wang, Wayne M Best, Catherine J Perez, Scott MJ Cornwall, Sarah K Keatley, RC Andrew Thompson, Susan A Charman, Karen L White, Eileen Ryan, Gong Chen, Jean-Robert Ioset, Thomas W von Geldern, Eric Chatelain. Selection and optimization of hits from a high-throughput phenotypic screen against Trypanosoma cruzi. Future Med Chem. **5** (15) 1733-1753 (2013) \[[doi:10.4155/fmc.13.139](http://doi.org/10.4155/fmc.13.139)]

## Deposited Set 9: 11th January 2013 - GSK TCAMS Dataset (hits from M. Tuberculosis phenotypic screening)

[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

The chemical structures and the generated data are hereby made public under Creative Commons' CC0 license: [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/) as a resource for anti-tuberculosis lead identification and basic research into the druggable genome of M. Tuberculosis.

[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to ChEMBL-NTD.

If you publish on, or wish to reference the GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Ballell, L., Bates, R. H., Young, R. J., Alvarez-Gomez, D., Alvarez-Ruiz, E., Barroso, V., Blanco, D., Crespo, B., Escribano, J., Gonzalez, R., Lozano, S., Huss, S., Santos-Villarejo, A., Martin-Plaza, J. J., Mendoza, A., Rebollo-Lopez, M. J., Remuinan-Blanco, M., Lavandera, J. L., Perez-Herran, E., Gamo-Benito, F. J., Garcia-Bustos, J. F., Barros, D., Castro, J. P. and Cammack, N. (2013), Fueling Open-Source Drug Discovery: 177 Small-Molecule Leads against Tuberculosis. ChemMedChem. \[[doi:10.1002/cmdc.201200428](http://dx.doi.org/10.1002/cmdc.201200428)]

## Deposited Set 8: 29th November 2012 - Identification of compounds with anti-proliferative activity against Trypanosoma brucei brucei BS427 by a whole cell viability based HTS campaign

[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the DNDi T.b. brucei dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Melissa L. Sykes, Jonathan B. Baell, Marcel Kaiser, Eric Chatelain, Danny Ganame, Jean-Robert Ioset, Vicky M Avery. Identification of compounds with anti-proliferative activity against Trypanosoma brucei brucei BS427 by a whole cell viability based HTS campaign. PLoS Negl Trop Dis. **6** (11) e1896 (2012) \[[doi:10.1371/journal.pntd.0001896](http://dx.doi.org/10.1371/journal.pntd.0001896)]

## Deposited Set 7: 9th October 2012 - The Harvard Medical School Liver Stage Malaria Dataset

[Harvard Medical School](http://hms.harvard.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the Harvard Medical School Liver Stage Malaria dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Emily R. Derbyshire, Miguel Prudencio, Maria M. Mota and Jon Clardy. Liver-stage malaria parasites vulnerable to diverse chemical scaffolds. PNAS **109** (22) 8511-8516 (2012) \[[abstract](http://www.pnas.org/content/109/22/8511.abstract)]

## Deposited Set 6: 18th November 2011 - Optimisation of fenarimol series for the treatment of Chagas disease, (DNDi Chagas Dataset) Partners: DNDi lead Optimization consortium (Epichem, Murdoch University, CDCO)

[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 5: 18th November 2011 - Screening and optimization of specific chemical series against human African Trypanosomiasis (HAT), (DNDi HAT Dataset) Partners: SCYNEXIS Inc. as part of the HAT Lead Optimisation consortium

[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 4: 18th November 2011 - Novartis-GNF Liver Stage Dataset (hits from P. yoelii liver stage screening)

[Novartis](http://www.novartis.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

[Novartis](http://www.novartis.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).

If you publish on, or wish to reference the Novartis-GNF dataset, please include the link to ChEMBL-NTD ([www.ebi.ac.uk/chemblntd](http://www.ebi.ac.uk/chemblntd)) and adapt the following citation language:

Stephan Meister, David M Plouffe, Kelli L. Kuhen, Ghislain M.C. Bonamy, Tao Wu, S. Whitney Barnes, Selina E. Bopp, Rachel Borboa, A. Taylor Bright, Jianwei Che, Steve Cohen, Neekesh V. Dharia, Kerstin Gagaring, Montip Gettayacamin, Perry Gordon, Todd Groessl, Nobutaka Kato, Marcus C.S. Lee, Case W. McNamara, David A. Fidock, Advait Nagle, Tae-gyu Nam, Wendy Richmond, Jason Roland, Matthias Rottmann, Bin Zhou, Patrick Froissard, Richard J. Glynne, Dominique Mazier, Jetsumon Sattabongkot, Peter G. Schultz, Tove Tuntland, John R. Walker, Yingyao Zhou, Arnab Chatterjee, Thierry T. Diagana, Elizabeth A. Winzeler. Imaging of Plasmodium liver stages to drive next-generation antimalarial drug discovery. \[[Science (2011)](http://dx.doi.org/10.1126/science.1211936)]

## Deposited Set 3: 20th May 2010 - St. Jude Children's Research Hospital Dataset

The [St. Jude Children's Research Hospital](http://www.stjude.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the St. Jude Children's Research Hospital dataset, please include the link to ChEMBL-NTD ([www.ebi.ac.uk/chemblntd](http://www.ebi.ac.uk/chemblntd)) and adapt the following citation language:

W. Armand Guiguemde, Anang A. Shelat, David Bouck, Sandra Duffy, Gregory J. Crowther, Paul H. Davis, David C. Smithson, Michele Connelly, Julie Clark, Fangyi Zhu, Maria B. Jimenez-Diaz, Maria S. Martinez, Emily B. Wilson, Abhai K. Tripathi, Jiri Gut, Elizabeth R. Sharlow, Ian Bathurst, Farah El Mazouni, Joseph W. Fowble, Isaac Forquer, Paula L. McGinley, Steve Castro, Inigo Angulo-Barturen, Santiago Ferrer, Philip J. Rosenthal, Joseph L. DeRisi, David J. Sullivan, John S. Lazo, David S. Roos, Michael K. Riscoe, Margaret A. Phillips, Pradipsinh K. Rathod, Wesley C. Van Voorhis, Vicky M. Avery and R. Kiplin Guy. Chemical genetics of Plasmodium falciparum. Nature **465** (7296) 311-315 (2010) \[[pdf](http://www.nature.com/nature/journal/v465/n7296/pdf/nature09099.pdf)]

## Deposited Set 2: 20th May 2010 - Novartis-GNF Malaria Box Dataset (hits from P. falciparum whole-cell screening)

[Novartis](http://www.novartis.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

[Novartis](http://www.novartis.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).

If you publish on, or wish to reference the Novartis-GNF dataset, please include the link to [ChEMBL-NTD ](https://chembl.gitbook.io/chembl-ntd/)and adapt the following citation language:

Novartis-GNF Malaria Box, K Gagaring, R Borboa, C Francek, Z Chen, J Buenviaje, D Plouffe, E Winzeler, A Brinker, T Diagana, J Taylor, R Glynne, A Chatterjee, K Kuhen. Genomics Institute of the Novartis Research Foundation (GNF), 10675 John Jay Hopkins Drive, San Diego CA 92121, USA and Novartis Institute for Tropical Disease, 10 Biopolis Road, Chromos # 05-01, 138 670 Singapore

## Deposited Set 1: 20th May 2010 - GSK TCAMS Dataset (hits from P. falciparum whole-cell screening)

[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

The chemical structures and the generated data are hereby made public under Creative Commons' CC0 license: [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/) as a resource for antimalarial lead identification and basic research into the druggable genome of P. falciparum.

[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).

If you publish on, or wish to reference the GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

Francisco-Javier Gamo, Laura M. Sanz, Jaume Vidal, Cristina de Cozar, Emilio Alvarez, Jose-Luis Lavandera, Dana E. Vanderwall, Darren V. S. Green, Vinod Kumar, Samiul Hasan, James R. Brown, Catherine E. Peishoff, Lon R. Cardon and Jose F. Garcia-Bustos. Thousands of chemical starting points for antimalarial lead identification. Nature **465** (7296) 305-310 (2010) \[[pdf](http://www.nature.com/nature/journal/v465/n7296/pdf/nature09107.pdf)]

## Useful Resources

PlasmoDB: [http://plasmodb.org/plasmo/](http://plasmodb.org/plasmo/)\
EuPathDB: [http://eupathdb.org/eupathdb/](http://eupathdb.org/eupathdb/)\
EnsemblProtists: [http://protists.ensembl.org/index.html](http://protists.ensembl.org/index.html)\
GeneDB: [http://www.genedb.org/Homepage](http://www.genedb.org/Homepage)\
Sanger Centre Plasmodium Genome Projects: [http://www.sanger.ac.uk/Projects/P\_falciparum/](http://www.sanger.ac.uk/Projects/P\_falciparum/)\
Medicines for Malaria Ventures: [http://www.mmv.org](http://www.mmv.org/)\
Drugs for Neglected Diseases Initiative: [http://www.dndi.org/](http://www.dndi.org/)\
The Bill and Melinda Gates Foundation [http://www.gatesfoundation.org/Pages/home.aspx](http://www.gatesfoundation.org/Pages/home.aspx)\
St. Jude Children's Research Hospital: [http://www.stjuderesearch.org](http://www.stjuderesearch.org/)\
Novartis Institute Tropical Disease: [http://www.novartis.com/research/nitd/index.shtml](http://www.novartis.com/research/nitd/index.shtml)\
TDR Targets: [http://www.tdrtargets.org](http://www.tdrtargets.org/)\
ChEMBLdb: [https://www.ebi.ac.uk/chembl/beta/](https://www.ebi.ac.uk/chembl/beta/)\
ChEMBL Blog: [http://chembl.blogspot.com/](http://chembl.blogspot.com/)

[![](https://www.ebi.ac.uk/chemblntd/static/images/general/wtlogo.png)](http://www.wellcome.ac.uk/) [![](https://www.ebi.ac.uk/chemblntd/static/images/general/mmv-logo.png)](http://www.mmv.org/) ![spacer](https://www.ebi.ac.uk/inc/images/spacer.gif)
