# ChEMBL-NTD

Welcome to the **ChEMBL - Neglected Tropical Disease** archive, a repository for Open Access primary screening and medicinal chemistry data directed at neglected diseases - endemic tropical diseases of the developing regions of the Africa, Asia, and the Americas. The primary purpose of ChEMBL-NTD is to provide a freely accessible and permanent archive and distribution centre for deposited data. ChEMBL-NTD is a subset of the data in the free medicinal chemistry and drug discovery database [ChEMBL](http://www.ebi.ac.uk/chembl/beta/). We actively encourage download and use of the [data](downloads/).

ChEMBL-NTD is maintained by EMBL-EBI at Hinxton in the United Kingdom, and has received no funding from any commercial organisation, individual or investor. More information on the EMBL-EBI can be found [here](http://www.ebi.ac.uk/).

The deposited data are hosted on the servers of the EMBL-EBI at Hinxton in the United Kingdom. The static pages are hosted by [GitBook](https://www.gitbook.com/).

If you have questions about the data - [contact us!](mailto:prudence@ebi.ac.uk?subject=[ChEMBL-NTD]%20Data%20query)

If you wish to deposit data - [contact us!](mailto:prudence@ebi.ac.uk?subject=[ChEMBL-NTD]%20Contribute%20data%20query%20link)

## ChEMBL-NTD Terms of Use

We encourage all users to download, copy and redistribute these data as needed. However, in the spirit of open collaboration and to enable rapid development of new therapeutics for neglected disease, we encourage following these basic principles:

* Users who annotate, add to, or modify these data in a way that adds significant value are encouraged to release their work to the public domain, ideally by re-contributing their findings to ChEMBL-NTD.
* When these data are used or cited in a paper or other scholarly work please reference the citation provided in each deposition set.

Access to the ChEMBL-NTD data is under the [EMBL-EBI's standard Terms of Use](http://www.ebi.ac.uk/Information/termsofuse.html)

## Deposited Set 21: 14th October 2016 – MMV Pathogen Box

![MMV Pathogen Box](https://www.ebi.ac.uk/chemblntd/static/images/home/mmv_pathogen_box.png)

Welcome to first [Medicines for Malaria Venture](http://www.mmv.org/) Pathogen Box dataset. The Pathogen Box contains 400 diverse, drug-like molecules active against neglected diseases of interest. This dataset contains biological activity confirmation data performed in the following disease areas: Tuberculosis, Malaria, Chagas disease, Leishmaniasis, Human African Trypanosomiasis, Cryptosporidiosis, Lymphatic Filariasis, Onchocerciasis, Schistosomiasis, Dengue, Chikungunya, Toxoplasmosis.

[Medicines for Malaria Venture](http://www.mmv.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

If you publish on, or wish to reference the data please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd). List of contributors:

* Nathan group, Weill Cornell Medical College
* Barry Lab, NIAID NIH
* Winzeler lab, UCSD
* Laboratory of Microbiology, Parasitology and Hygiene \(LMPH\), Univ. of Antwerp
* London School of Hygiene & Tropical Medicine
* Parasite Chemotherapy Unit at Swiss Tropical and Public Health Institute
* Huston group at the University of Vermont
* Castellanos group at UTMB, Galveston, Texas
* Sakanari Lab, Univ. of Calif. San Francisco
* Townson \(Simon\) Lab, Imperial College London
* Fidelis Cho-Ngwa Lab, University of Buea, Buea, Cameroon
* Laboratory of Molecular Parasitology at the New York Blood Center
* Bickle Lab, London School of Hygiene & Tropical Medicine
* Keiser Lab, Swiss Tropical and Public Health Institute
* Caffrey group at the Center for Discovery and Innovation in Parasitic Diseases, Skaggs School of Pharmacy and Pharmaceutical Sciences, UC San Diego
* University of Cape Town
* Shanmugam group at CSIR-National Chemical Laboratory, Pune \(India\)
* Novartis Institute for Tropical Disease-Singapore
* Abbvie

## Deposited Set 20: 2nd August 2016 - SwissTPH T. brucei screening hits

![Swiss TPH Image](https://www.ebi.ac.uk/chemblntd/static/images/home/swiss_tph.png)

Welcome to the [SwissTPH](http://www.swisstph.ch/) T. brucei screening hits dataset. The protozoan parasites Trypanosoma brucei spp., the causative agents of human sleeping sickness, are transmitted by tsetse flies. During differentiation from bloodstream forms to procyclic forms, which naturally occur in the midgut of the fly, the trypanosomes shed their coat of variant surface glycoprotein \(VSG\). This process would be lethal for the parasites while in the bloodstream of an infected patient. We searched in a medium-throughput screen for triggers that artificially stimulate differentiation towards procyclic forms at 37°C and identified 28 hits. The differentiation-inducing hits could facilitate studies on the regulation of differentiation and serve as lead scaffolds for medicinal chemistry in order to develop new treatments for sleeping sickness.   
  
[SwissTPH](http://www.swisstph.ch/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the data please include the link to [ChEMBL-NTD](https://www.ebi.ac.uk/chemblntd) and adapt the following citation language: Wenzler T, Schumann Burkard G, Schmidt RS, Mäser P, Bergner A, Roditi I, Brun R. A new approach to chemotherapy: drug-induced differentiation kills African trypanosomes. Sci Rep. 2016 Mar 2;6:22451. [doi:10.1038/srep22451](http://www.nature.com/articles/srep22451). 

## Deposited Set 19: 5th March 2016 - UW kinase screening hits

![University of Washington Image](https://www.ebi.ac.uk/chemblntd/static/images/home/UoW.png)  
Welcome to the [UW Crowther Lab](http://faculty.washington.edu/crowther/) kinase screening hits dataset. We screened ~13,000 cell-active compounds for activity against five different protein kinases. GlaxoSmithKline’s [Tres Cantos Antimalarial Set \(TCAMS\)](https://www.ebi.ac.uk/chemblntd/download/#tcams_dataset) was screened in biochemical assays of Plasmodium falciparum calcium-dependent protein kinases 1 and 4 \(CDPK1 and CDPK4\), mitogen-associated protein kinase 2 \(MAPK2/MAP2\), protein kinase 6 \(PK6\), and protein kinase 7 \(PK7\). Novel potent inhibitors \(IC50 &lt; 1 uM\) were discovered for three of the kinases: CDPK1, CDPK4, and PK6. The PK6 inhibitors are the most potent yet discovered for this enzyme and deserve further scrutiny.   
  
[UW](http://www.washington.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the UW kinase screening dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: 

Gregory J. Crowther, Heidi K. Hillesland, Katelyn R. Ceylon, Molly C. Reid, Maria Jose Lafuente-Monasterio, Sonja Ghidelli-Disse, Stephen E. Leonard, Panqing He, Jackson C. Jones, Mallory M. Krahn, Jack S. Mo, Kartheek S. Dasari, Anna M. W. Fox, Markus Boesche, Majida El Bakkouri, Kasey L. Rivas, Didier Leroy, Raymond Hui, Gerard Drewes, Dustin J. Maly, Wesley C. Van Voorhis, Kayode K. Ojo. Biochemical Screening of Five Protein Kinases from Plasmodium falciparumagainst 14,000 Cell-Active Compounds. PLoS ONE, 2016. [doi:10.1371/journal.pone.0149996](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0149996). 

## Deposited Set 18: 15th February 2016 - UCSD Winzeler Lab \(hits from exoerytrocytic-stage screening\)

![UC San Diego School of Medicine Image](https://www.ebi.ac.uk/chemblntd/static/images/home/sandiego_scm.png)

Welcome to the [UC San Diego School of Medicine Winzeler Lab](http://winzeler.ucsd.edu/) exoerytrocytic-stage screening hits dataset. We have developed a luciferase-based phenotypic screen of malaria exoerythrocytic-stage parasites optimized for a 1536-well format. This assay uses the exoerythrocytic-stage of the rodent malaria parasite, Plasmodium berghei, and a human hepatoma cell line. We use this assay to evaluate several biased and unbiased compound libraries, including two small sets of molecules \(400 and 89 compounds, respectively\) with known activity against malaria erythrocytic-stage parasites and a set of 9,886 Diversity-Oriented Synthesis \(DOS\)-derived compounds. Of the compounds screened we obtain hit rates of 12-13% and 0.6% in preselected and naïve libraries, respectively, and identify 52 compounds with exoerythrocytic-stage activity less than 1mM, and having minimal host cell toxicity. Our data demonstrate the ability of this method to identify compounds known to have causal prophylactic activity in both human and animal models of malaria, as well as novel compounds, including several exclusively active against parasite exoerythrocytic stages.   
  
[UCSD](https://ucsd.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the UCSD exoerythrocytic-stage dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: 

Justine Swann, Victoria Corey, Christina A. Scherer, Nobutaka Kato, Eamon Comer, Micah Maetani, Yevgeniya Antonova-Koch, Christin Reimer, Kerstin Gagaring, Maureen Ibanez, David Plouffe, Anne-Marie Zeeman, Clemens H. M. Kocken, Case W. McNamara, Stuart L. Schreiber, Brice Campo, Elizabeth A. Winzeler, Stephan Meister. A high-throughput luciferase-based assay for the discovery of therapeutics that prevent malaria. ACS Infectious Diseases, 2016. [doi:10.1021/acsinfecdis.5b00143](http://pubs.acs.org/doi/abs/10.1021/acsinfecdis.5b00143). 

## Deposited Set 17: 25th January 2016 - 2nd Tres Cantos TB TCAMS dataset \(hits from M. Tuberculosis phenotypic screening\)

![TCAMS Image](https://www.ebi.ac.uk/chemblntd/static/images/home/tcams.png)  
Welcome to the second Tres Cantos TB TCAMS dataset. As a follow up to the antimycobacterial screening effort and subsequent release of GSK´s first Tres Cantos Antimycobacterial Set \([Set 9, TCAMS-TB, deposited on the 11th of January 2013](https://www.ebi.ac.uk/chemblntd/download/#tcams_dataset_tb)\), a second antitubercular screening campaign of 250,000 structurally novel compounds recently added to the GSK collection was performed. The compounds were further prioritized based on, not only antitubercular potency, but also on their physicochemical characteristics. The results for 50 most attractive compounds \(Activity against BCG, Replicating and Non-replicating Mtb, Intracellular activity and against resistant strains\) are made available in this deposition, as well as a number of critical physico-chemical properties. All efforts have been made to ensure data quality and accuracy, but users are reminded that these data carry the usual caveats associated with results from large-scale screening.   
  
[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the second GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/)  and adapt the following citation language:   
  
Rebollo-Lopez, M. J.; Lelièvre, J.; Alvarez-Gomez, D.; Castro-Pichel, J.; Martínez-Jiménez, F.; Papadatos, G.; Kumar, V.; Colmenarejo, G.; Mugumbate, G.; Hurle, M.; Barroso, V.; Young, R. J.; Martinez-Hoyos, M.; González del Río, R.; Bates, R. H.; Lopez-Roman, E. M.; Mendoza-Losana, A.; Brown, J. R.; Alvarez-Ruiz, E.; Marti-Renom, M. A.; Overington, J. P.; Cammack, N.; Ballell, L.; Barros-Aguire, D., Release of 50 new, drug-like compounds and their computational target predictions for open source anti-tubercular drug discovery. PLoS ONE 2015, 10, e0142293. \[[doi:10.1371/journal.pone.0142293 ](http://dx.doi.org/10.1371/journal.pone.0142293)\]

## Deposited Set 16: 25th November 2015 - TropIQ SMFA Dataset

![TropiqHS Image](https://www.ebi.ac.uk/chemblntd/static/images/home/tropiqhs.png)  
Welcome to the [TropIQ Health Sciences](http://tropiq.nl/) SMFA \(Standard Membrane Feeding Assay\) dataset. This dataset has been generated in the context of the ‘gametocyte assays for malaria’ effort initiated by the [Bill & Melinda Gates Foundation](http://www.gatesfoundation.org/). In this effort, five consortia each comprised of two to five laboratories have screened compounds against gametocytes, the transmission stages of the malaria parasite. TropIQ has subsequently evaluated 18 compounds selected by the consortia in mosquito feeding assays \(SMFA\), in order to evaluate their transmission-blocking effect and to provide validation data to malaria research community. The compounds are part of the [MMV malaria box](http://www.mmv.org/malariabox) and were tested at six doses in duplicate.   
  
[TropIQ](http://tropiq.nl/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use. 

## Deposited Set 15: 18th August 2015 - DNDi Dataset: Antiprotozoal activity profiling of approved drugs

![DNDi Image](https://www.ebi.ac.uk/chemblntd/static/images/home/dndi.png)  
In this study, a set of 100 registered drugs with drug repositioning potential for neglected tropical diseases was assembled. The compound collection was systematically screened against protozoan parasites, namely T. b. rhodesiense, L. donovani, T. cruzi and P. falciparum. Several drugs and drug classes exhibited in vitro activity and selectivity against one of the protozoan parasites. The results offer opportunities for drug repurposing and the identified compound classes could also be a starting point for new drug discovery projects.   
  
[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the DNDi drug repositioning dataset please include the link to ChEMBL-NTD and adapt the following citation language: Kaiser M, Mäser P, Tadoori LP, Ioset JR, Brun R. Antiprotozoal Activity Profiling of Approved Drugs: A Starting Point toward Drug Repositioning. PLoS One. 2015 10\(8\): e0135556. \[[doi:10.1371/journal.pone.0135556](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0135556)\] 

## Deposited Set 14: 5th March 2015 - GSK TCAKS Dataset \(hits from Leishmania donovani, Trypanosoma cruzi and Trypanosoma brucei brucei phenotypic screening\)

![TCAMS Image](https://www.ebi.ac.uk/chemblntd/static/images/home/tcams.png)  
Welcome to the GSK TCAKS dataset. Using whole-cell phenotypic assays, the GSK screening collection of 1.8 million compounds was screened against the three kinetoplastids most relevant to human disease, i.e. Leishmania donovani, Trypanosoma cruzi and Trypanosoma brucei. Secondary confirmatory and orthogonal intracellular anti-parasiticidal assays were conducted, and the potential for non-specific cytotoxicity determined with HepG2 assay. Hit compounds were chemically clustered and triaged for desirable physicochemical properties. Consequently, three anti-kinetoplastid chemical boxes of 222 compounds for Chagas-Box, 192 compounds for Leish-Box and 192 compounds for HAT-Box were assembled. The compound sets are provided as an open resource for future lead discovery programs, as well as to address important research questions.   
  
[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to ChEMBL-NTD.   
  
If you publish on, or wish to reference the GSK TCAKS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

 Imanol Pena, M. Pilar Manzano, Juan Cantizani, Albane Kessler, Julio Alonso-Padilla, Ana I. Bardera, Emilio Alvarez, Gonzalo Colmenarejo, Ignacio Cotillo, Irene Roquero, Francisco de Dios-Anton, Vanessa Barroso, Ana Rodriguez, David W. Gray, Miguel Navarro, Vinod Kumar, Alexander Sherstnev, David Drewry, James R. Brown, Jose M. Fiandor & J. Julio Martin. \(2015\), New Compound Sets Identified from High Throughput Phenotypic Screening against Three Kinetoplastid Parasites: An Open Resource. \[doi:[10.1038/srep08771](http://dx.doi.org/10.1038/srep08771)\] 

## Deposited Set 13: 4th February 2015 - St. Jude Children's Research Hospital Dataset

![St. Judes GNF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/stjudes.png)  
Welcome to the [St. Jude Children's Research Hospital](http://www.stjude.org/) NTD dataset. The data deposited were derived from a screen of the TCAMS and Malaria Box libraries for compounds that selectively inhibit the [Plasmodium falciparum hexose transporter, PfHT ](https://www.ebi.ac.uk/chembl/target/inspect/CHEMBL4697), versus the [human glucose transporter GLUT1](https://www.ebi.ac.uk/chembl/target/inspect/CHEMBL2535). Both PfHT and GLUT1 were expressed heterologously in a glucose transporter null mutant of the protozoan Leishmania mexicana \(Δlmxgt1-3 null mutant; expression system described [here](http://www.ncbi.nlm.nih.gov/pubmed/18708094)\). When grown in the appropriate medium, these transgenic Leishmaniareporter strains depend upon uptake of glucose for growth, and thus compounds that inhibit the heterologously expressed hexose transporters will prevent growth of the reporter strain. Growth was monitored using a SYBR Green fluorescence assay that detected parasite DNA. One set of data reports the growth inhibition effect of each compound in the TCAMS library \(13,243 compounds\), at 2 uM concentration, against Δlmxgt1-3 parasites expressing PfHT, GLUT1, or the L. mexicana glucose transporter LmxGT2. The second set of data report the dose-response data for 392 selected TCAMS compounds against strains expressing each of the 3 heterologously expressed hexose transporters. The third set of data reports dose-response data for all 400 compounds of the Malaria Box library against the same 3 reporter strains.   
  
The [St. Jude Children's Research Hospital](http://www.stjude.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.

## Deposited Set 12: 11th October 2014 - Antimalarial screening hits dataset

![UCSF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/mmv.png)  
Welcome to the [MMV](http://www.mmv.org/) antimalarial screening hits [dataset](ftp://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLNTD/set12_MMV/). A selection cascade has been applied to prioritize hits resulting from the screening of a diverse chemical library against blood-stage P. falciparum. The cascade led to the selection of 15 novel chemical series from which ten confirmed their activity when newly synthesized samples were tested. The hits will be fully profiled in additional Plasmodium assays, as well as in in vitro distribution metabolism pharmacokinetic \(DMPK\) assays. In particular, MMV is prosecuting most of the series and encourages research groups to contact them should they be interested in these hits.   
  
[MMV](http://www.mmv.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the MMV dataset set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Avery, V. M., Bashyam, S., Burrows, J. N., Duffy, S., Papadatos, G., Puthukkuti, S., et al. \(2014\). Screening and hit evaluation of a chemical library against blood-stage Plasmodium falciparum. Malaria Journal, 13, 190. \[[doi:10.1186/1475-2875-13-190](http://dx.doi.org/10.1186/1475-2875-13-190)\] 

## Deposited Set 11: 12th November 2013 - UCSF DeRisi Lab MMV Dataset

![UCSF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/ucsf.png)  
Welcome to the [UC San Francisco DeRisi Lab](http://derisilab.ucsf.edu/) Malaria Box apicoplast specific dataset. Work in the DeRisi lab has recently established that the only essential apicoplast metabolic function during asexual blood phase growth is isoprenoid synthesis \([Yeh and DeRisi, PLoS Biol. 2011](http://doi.org/10.1371/journal.pbio.1001138)\). Cultures supplemented with the sole essential metabolic output of this pathway, isopentenyl pyrophosphate \(IPP\), are completely resistant to antibiotic treatment and moreover, they continue to grow and propagate indefinitely even though these parasites are devoid of the apicoplast genome and organelle \(apicoplast\(-\)\). Thus, rescue with IPP provides an unique opportunity to determine whether the specific action of anti-malarial compound is targeted at the apicoplast. The DeRisi lab screened the MMV Malaria Box set of 400 anti-malarial compounds using the W2 strain in a 72 hr growth assay monitored by flow cytometry both in the presence and absence of supplemental IPP. Here, we report the raw screening data for growth in both conditions when treated with 5uM of each Malaria Box compound.   
  
This work was funded by a [Grand Challenges Exploration grant from the ](http://www.grandchallenges.org/Explorations/Pages/Introduction.aspx)[Bill and Melinda Gates Foundation](http://www.gatesfoundation.org/) and [The Howard Hughes Medical Institute](http://www.hhmi.org/).   
  
UC San Francisco and the Howard Hughes Medical Institute do not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use. 

## Deposited Set 10: 28th October 2013 - Selection and optimization of hits from a high-throughput phenotypic screen against Trypanosoma cruzi

![DNDi Image](https://www.ebi.ac.uk/chemblntd/static/images/home/dndi.png)  
Welcome to the [DNDi](http://www.dndi.org/) T. cruzi Dataset. Inhibitors of Trypanosoma cruzi with novel mechanisms of action are urgently required to diversify the current clinical and preclinical pipelines. Increasing the number and diversity of hits available for assessment at the beginning of the discovery process will help to achieve this aim. Results: We report the evaluation of multiple hits generated from a high-throughput screen to identify inhibitors of T. cruzi and from these studies the discovery of two novel series currently in lead optimization. Lead compounds from these series potently and selectively inhibit growth of T. cruzi in vitro and the most advanced compound is orally active in a subchronic mouse model of T. cruzi infection. Conclusion: High-throughput screening of novel compound collections has an important role to play in diversifying the trypanosomatid drug discovery portfolio. A new T. cruzi inhibitor series with good drug-like properties and promising in vivo efficacy has been identified through this process.   
  
**15th April 2014** - Two additional T. cruzi datasets have been deposited by DNDi. The two new fenarimol series datasets, from which preclinical candidates EPL-BS0967 and EPL-BS1246 were identified, are available to download [here](downloads/#deposited-set-8-dndi-t-b-brucei-dataset-29-th-november-2012).  
  
[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the DNDi T. cruzi dataset please include the link to  [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: 

Martine Keenan, Paul W. Alexander, Jason H Chaplin, Michael J Abbott, Hugo Diao, Zhisen Wang, Wayne M Best, Catherine J Perez, Scott MJ Cornwall, Sarah K Keatley, RC Andrew Thompson, Susan A Charman, Karen L White, Eileen Ryan, Gong Chen, Jean-Robert Ioset, Thomas W von Geldern, Eric Chatelain. Selection and optimization of hits from a high-throughput phenotypic screen against Trypanosoma cruzi. Future Med Chem. **5** \(15\) 1733-1753 \(2013\) \[[doi:10.4155/fmc.13.139](http://doi.org/10.4155/fmc.13.139)\] 

## Deposited Set 9: 11th January 2013 - GSK TCAMS Dataset \(hits from M. Tuberculosis phenotypic screening\)

![TCAMS Image](https://www.ebi.ac.uk/chemblntd/static/images/home/tcams.png)  
Welcome to the Tres Cantos TB TCAMS dataset. The results of a 2-million-compound anti-mycobacterial phenotypic screening campaign against Mycobacterium Bovis BCG with hit confirmation in M. Tuberculosis H37Rv are made publicly accessible. 776 hits against BCG including 177 non-cytotoxic H37Rv potent hits were identified and are made available here. All efforts have been made to ensure data quality and accuracy, but users are reminded that these data carry the usual caveats associated with results from large scale screening.   
  
[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
The chemical structures and the generated data are hereby made public under Creative Commons' CC0 license: [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/) as a resource for anti-tuberculosis lead identification and basic research into the druggable genome of M. Tuberculosis.   
  
[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to ChEMBL-NTD.  
  
If you publish on, or wish to reference the GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: 

Ballell, L., Bates, R. H., Young, R. J., Alvarez-Gomez, D., Alvarez-Ruiz, E., Barroso, V., Blanco, D., Crespo, B., Escribano, J., Gonzalez, R., Lozano, S., Huss, S., Santos-Villarejo, A., Martin-Plaza, J. J., Mendoza, A., Rebollo-Lopez, M. J., Remuinan-Blanco, M., Lavandera, J. L., Perez-Herran, E., Gamo-Benito, F. J., Garcia-Bustos, J. F., Barros, D., Castro, J. P. and Cammack, N. \(2013\), Fueling Open-Source Drug Discovery: 177 Small-Molecule Leads against Tuberculosis. ChemMedChem. \[[doi:10.1002/cmdc.201200428](http://dx.doi.org/10.1002/cmdc.201200428)\] 

## Deposited Set 8: 29th November 2012 - Identification of compounds with anti-proliferative activity against Trypanosoma brucei brucei BS427 by a whole cell viability based HTS campaign

![DNDi Image](https://www.ebi.ac.uk/chemblntd/static/images/home/dndi.png)  
Welcome to the [DNDi](http://www.dndi.org/) T.b. brucei Dataset. Human African Trypanosomiasis \(HAT\) is caused by two trypanosome sub-species, Trypanosoma brucei rhodesiense and Trypanosoma brucei gambiense. Current drugs available for the treatment of HAT have associated, and significant, issues related to difficult administration regimes and limited effectiveness across species and disease stages. There is a considerable need to find new alternative and less toxic drugs. A well recognised approach to identify starting points for new drug candidates is high throughput screening \(HTS\) of large compound library collections. We describe the application of an Alamar Blue based, 384-well HTS assay to screen a library of 87,926 compounds against the related trypanosome subspecies, Trypanosoma brucei brucei BS427. The assay was shown to be reproducible throughout the screening of the 248 compound plates, undertaken over an 11 day period, with control plates exhibiting a mean Z' of 0.7. Primary hits identified against T.b. brucei were retested and the IC50 value for each compound was estimated for both T.b. brucei and a mammalian cell line HEK293, to determine a selectivity index for each compound. The screening campaign identified 205 compounds with greater than 10 times selectivity against T.b. brucei. Cluster analysis of these compounds, taking into account the required chemical and drug-like structural properties, afforded a panel of eight compounds for further biological analysis. These compounds had IC50 values ranging from 0.22 M to 4 M with associated selectivity indices ranging from 19 to greater than 345. Further testing against T.b. rhodesiense led to the selection of 6 compounds from 5 new classes with activity against the causative species of HAT which can be considered potential candidates for HAT early drug discovery.   
  
[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the DNDi T.b. brucei dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Melissa L. Sykes, Jonathan B. Baell, Marcel Kaiser, Eric Chatelain, Danny Ganame, Jean-Robert Ioset, Vicky M Avery. Identification of compounds with anti-proliferative activity against Trypanosoma brucei brucei BS427 by a whole cell viability based HTS campaign. PLoS Negl Trop Dis. **6** \(11\) e1896 \(2012\) \[[doi:10.1371/journal.pntd.0001896](http://dx.doi.org/10.1371/journal.pntd.0001896)\] 

## Deposited Set 7: 9th October 2012 - The Harvard Medical School Liver Stage Malaria Dataset

![Havard Image](https://www.ebi.ac.uk/chemblntd/static/images/home/harvard.png)  
Welcome to the [Harvard Medical School](http://hms.harvard.edu/) Liver Stage Malaria Dataset. A high-throughput phenotypic liver-stage Plasmodium parasite screen was developed to systematically identify molecules with liver-stage efficacy. The screen recapitulates liver-stage infection by isolating luciferase- expressing P. berghei parasites directly from the salivary glands of infected mosquitoes, adding them to confluent human liver cells in 384-well plates, and measuring luciferase activity after a suitable incubation period. Screening 5,375 known bioactive compounds identified 37 liver-stage malaria inhibitors with diverse modes of action.   
  
[Harvard Medical School](http://hms.harvard.edu/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the Harvard Medical School Liver Stage Malaria dataset please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: Emily R. Derbyshire, Miguel Prudencio, Maria M. Mota and Jon Clardy. Liver-stage malaria parasites vulnerable to diverse chemical scaffolds. PNAS **109** \(22\) 8511-8516 \(2012\) \[[abstract](http://www.pnas.org/content/109/22/8511.abstract)\] 

## Deposited Set 6: 18th November 2011 - Optimisation of fenarimol series for the treatment of Chagas disease, \(DNDi Chagas Dataset\) Partners: DNDi lead Optimization consortium \(Epichem, Murdoch University, CDCO\)

![DNDi Image](https://www.ebi.ac.uk/chemblntd/static/images/home/dndi.png)  
Welcome to the [DNDi](http://www.dndi.org/) Chagas Dataset. As part of a hit generation effort to discover new T. cruzi inhibitors, a diversity set of agrochemicals was screened in vitro in a whole cell parasite assay against a T. cruzi strain \(Tulahuen\) belonging to T. cruzi group VI \(TcVI\) \[1\] and transfected with the Lac-Z gene. \[2\] The plant fungicides fenarimol \[DNDI1246778\] and pyrifenox \[DNDI1255613\], used in the control of powdery mildew and fungus on fruits and vegetables, demonstrated good activity in the assay with IC50's of 350 nM and 290 nM, respectively. With simple aromatic components and modular synthesis, fenarimol was an ideal starting point for structure-activity-relationship \(SAR\) investigations aimed at improving activity against T. cruzi and developing compounds suitable for in vivo characterisation.   
  
[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use. 

## Deposited Set 5: 18th November 2011 - Screening and optimization of specific chemical series against human African Trypanosomiasis \(HAT\), \(DNDi HAT Dataset\) Partners: SCYNEXIS Inc. as part of the HAT Lead Optimisation consortium

![DNDi Image](https://www.ebi.ac.uk/chemblntd/static/images/home/dndi.png)  
Welcome to the [DNDi](http://www.dndi.org/) Human African Trypanosomiasis \(HAT\) dataset. The majority of the 4926 compounds released in this dataset from DNDi were identified by screening of combinatorial libraries of two structural classes, 2-aryl-5-aminomethyloxazoles and N-substituted pyridylamidoximes, prepared by SCYNEXIS, Inc. for in vitro activity against Trypanosoma brucei. Based on activity observed in these screening activities, Hit-to-Lead and Lead Optimization programs were conducted opposite these two series and additional SAR was developed for in vitro anti-parasitic activity, physicochemical and in vitro ADME properties. Work in both series was suspended upon determination that the structural requirements for good in vitro anti-parasitic activity were inconsistent with structural requirements for good potential for in vivo activity. More specifically, in both series, compounds with good in vitro anti-parasitic activity tended to be lipophilic, metabolically unstable and of limited aqueous solubility. In addition to these two series, a number of small sets of compounds based on literature reports of in vitro parasitic activity are reported. For example, a series of pyrazolopyrimidines is described. These compounds were found to be poorly selective for in vitro anti-parasitic activity relative to cytotoxicity to mammalian cells, hence were not further pursued.   
  
[DNDi](http://www.dndi.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use. 

## Deposited Set 4: 18th November 2011 - Novartis-GNF Liver Stage Dataset \(hits from P. yoelii liver stage screening\)

![Novartis GNF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/gnf.png)  
Welcome to the [Novartis-GNF](http://www.gnf.org/) Malaria Liver Stage dataset. We re-screened the blood stage inhibitors of the Malaria Box dataset 2 and identified chemical scaffolds with potent activity against both blood and liver stages. Of the 4422 compounds tested, 339 caused a &gt;50% reduction in P. yoelii schizont area. Of those 339 compounds, 275 could be confirmed as active against liver stage parasites \(fitted IC50 &lt; 10 uM\) in a second round of screening. Of these, 229 compounds had an IC50 of less than 1 uM and 86 less than 100 nM.   
  
[Novartis](http://www.novartis.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
[Novartis](http://www.novartis.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).  
  
If you publish on, or wish to reference the Novartis-GNF dataset, please include the link to ChEMBL-NTD \([www.ebi.ac.uk/chemblntd](http://www.ebi.ac.uk/chemblntd)\) and adapt the following citation language: 

Stephan Meister, David M Plouffe, Kelli L. Kuhen, Ghislain M.C. Bonamy, Tao Wu, S. Whitney Barnes, Selina E. Bopp, Rachel Borboa, A. Taylor Bright, Jianwei Che, Steve Cohen, Neekesh V. Dharia, Kerstin Gagaring, Montip Gettayacamin, Perry Gordon, Todd Groessl, Nobutaka Kato, Marcus C.S. Lee, Case W. McNamara, David A. Fidock, Advait Nagle, Tae-gyu Nam, Wendy Richmond, Jason Roland, Matthias Rottmann, Bin Zhou, Patrick Froissard, Richard J. Glynne, Dominique Mazier, Jetsumon Sattabongkot, Peter G. Schultz, Tove Tuntland, John R. Walker, Yingyao Zhou, Arnab Chatterjee, Thierry T. Diagana, Elizabeth A. Winzeler. Imaging of Plasmodium liver stages to drive next-generation antimalarial drug discovery. \[[Science \(2011\)](http://dx.doi.org/10.1126/science.1211936)\]

## Deposited Set 3: 20th May 2010 - St. Jude Children's Research Hospital Dataset

![Novartis GNF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/stjudes.png)  
Welcome to the [St. Jude Children's Research Hospital](http://www.stjude.org/) Malaria dataset. An international team led by St. Jude Children's Research Hospital investigators released data detailing the effectiveness of nearly 310,000 chemicals against a malaria parasite that remains one of the world's leading killers of young children. The research, which appears in the 20th May edition of the scientific journal Nature, identified more than 1,100 new compounds with confirmed activity against the malaria parasite. Of those, 172 were studied in detail; leading to identification of almost two dozen families of molecules investigators consider possible candidates for drug development.   
  
The [St. Jude Children's Research Hospital](http://www.stjude.org/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
If you publish on, or wish to reference the St. Jude Children's Research Hospital dataset, please include the link to ChEMBL-NTD \([www.ebi.ac.uk/chemblntd](http://www.ebi.ac.uk/chemblntd)\) and adapt the following citation language: 

W. Armand Guiguemde, Anang A. Shelat, David Bouck, Sandra Duffy, Gregory J. Crowther, Paul H. Davis, David C. Smithson, Michele Connelly, Julie Clark, Fangyi Zhu, Maria B. Jimenez-Diaz, Maria S. Martinez, Emily B. Wilson, Abhai K. Tripathi, Jiri Gut, Elizabeth R. Sharlow, Ian Bathurst, Farah El Mazouni, Joseph W. Fowble, Isaac Forquer, Paula L. McGinley, Steve Castro, Inigo Angulo-Barturen, Santiago Ferrer, Philip J. Rosenthal, Joseph L. DeRisi, David J. Sullivan, John S. Lazo, David S. Roos, Michael K. Riscoe, Margaret A. Phillips, Pradipsinh K. Rathod, Wesley C. Van Voorhis, Vicky M. Avery and R. Kiplin Guy. Chemical genetics of Plasmodium falciparum. Nature **465** \(7296\) 311-315 \(2010\) \[[pdf](http://www.nature.com/nature/journal/v465/n7296/pdf/nature09099.pdf)\]

## Deposited Set 2: 20th May 2010 - Novartis-GNF Malaria Box Dataset \(hits from P. falciparum whole-cell screening\)

![Novartis GNF Image](https://www.ebi.ac.uk/chemblntd/static/images/home/gnf.png)  
Welcome to the [Novartis-GNF](http://www.gnf.org/) Malaria Box dataset. Screening commercially available compounds in GNF's nonproprietary libraries identified inhibitors of proliferation of P. falciparum strain 3D7 in human erythrocytes. The dataset contains the structures and screening data for over 5,600 compounds, which were tested in dose response and confirmed to inhibit parasite growth by more than 50% at the highest screening concentration \(1.25 or 12.5 uM\). Over 3200 compounds of these compounds were available for testing in powder format which allowed for testing at higher concentrations \(12.5 uM\) using freshly diluted stocks. Activity against the multidrug resistant W2 strain is also available. In addition, we have included data for a human cell cytotoxicity selectivity screen using the Huh7 human hepatocellular carcinoma cell line and deposited an indication of the 'promiscuity' of the hits \(the IFI index\) in other high-throughput assays at the Genomics Institute of the Novartis Research Foundation. All efforts have been made to ensure data quality and accuracy, but users are reminded that these data carry the usual caveats associated with results from large scale screening.   
  
[Novartis](http://www.novartis.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
[Novartis](http://www.novartis.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).  
  
If you publish on, or wish to reference the Novartis-GNF dataset, please include the link to [ChEMBL-NTD ](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language: 

Novartis-GNF Malaria Box, K Gagaring, R Borboa, C Francek, Z Chen, J Buenviaje, D Plouffe, E Winzeler, A Brinker, T Diagana, J Taylor, R Glynne, A Chatterjee, K Kuhen. Genomics Institute of the Novartis Research Foundation \(GNF\), 10675 John Jay Hopkins Drive, San Diego CA 92121, USA and Novartis Institute for Tropical Disease, 10 Biopolis Road, Chromos \# 05-01, 138 670 Singapore

## Deposited Set 1: 20th May 2010 - GSK TCAMS Dataset \(hits from P. falciparum whole-cell screening\)

![TCAMS Image](https://www.ebi.ac.uk/chemblntd/static/images/home/tcams.png)  
Welcome to the Tres Cantos Antimalarial TCAMS dataset. Screening of approximately 2 million compounds in [GlaxoSmithKline's](http://www.gsk.com/) screening library identified inhibitors of proliferation of P. falciparum strain 3D7 in human erythrocytes. The dataset contains the structures and screening data for over 13,500 compounds confirmed to inhibit parasite growth by more than 80% at 2 uM concentration. The compounds' activity against the multidrug resistant Dd2 strain has also been measured for comparison. In addition, we have included data for a human cell cytotoxicity selectivity screen and also deposited an indication of the 'promiscuity' of the hits \(the IFI index\) in other high-throughput assays at GSK. Finally, a potential mode of action and predicted P. falciparum targets are listed for selected compounds. All efforts have been made to ensure data quality and accuracy, but users are reminded that these data carry the usual caveats associated with results from large scale screening.   
  
[GSK](http://www.gsk.com/) does not guarantee the accuracy of any data, nor the suitability of the data for any purpose, in accordance with the EBI Terms of Use.   
  
The chemical structures and the generated data are hereby made public under Creative Commons' CC0 license: [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/) as a resource for antimalarial lead identification and basic research into the druggable genome of P. falciparum.   
  
[GSK](http://www.gsk.com/) have committed to provide any corrections, additions and appropriate new annotations or data to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/).  
  
If you publish on, or wish to reference the GSK TCAMS set please include the link to [ChEMBL-NTD](https://chembl.gitbook.io/chembl-ntd/) and adapt the following citation language:

 Francisco-Javier Gamo, Laura M. Sanz, Jaume Vidal, Cristina de Cozar, Emilio Alvarez, Jose-Luis Lavandera, Dana E. Vanderwall, Darren V. S. Green, Vinod Kumar, Samiul Hasan, James R. Brown, Catherine E. Peishoff, Lon R. Cardon and Jose F. Garcia-Bustos. Thousands of chemical starting points for antimalarial lead identification. Nature **465** \(7296\) 305-310 \(2010\) \[[pdf](http://www.nature.com/nature/journal/v465/n7296/pdf/nature09107.pdf)\]

## Useful Resources

PlasmoDB: [http://plasmodb.org/plasmo/](http://plasmodb.org/plasmo/)  
EuPathDB: [http://eupathdb.org/eupathdb/](http://eupathdb.org/eupathdb/)  
EnsemblProtists: [http://protists.ensembl.org/index.html](http://protists.ensembl.org/index.html)  
GeneDB: [http://www.genedb.org/Homepage](http://www.genedb.org/Homepage)  
Sanger Centre Plasmodium Genome Projects: [http://www.sanger.ac.uk/Projects/P\_falciparum/](http://www.sanger.ac.uk/Projects/P_falciparum/)  
Medicines for Malaria Ventures: [http://www.mmv.org](http://www.mmv.org/)  
Drugs for Neglected Diseases Initiative: [http://www.dndi.org/](http://www.dndi.org/)  
The Bill and Melinda Gates Foundation [http://www.gatesfoundation.org/Pages/home.aspx](http://www.gatesfoundation.org/Pages/home.aspx)  
St. Jude Children's Research Hospital: [http://www.stjuderesearch.org](http://www.stjuderesearch.org/)  
Novartis Institute Tropical Disease: [http://www.novartis.com/research/nitd/index.shtml](http://www.novartis.com/research/nitd/index.shtml)  
TDR Targets: [http://www.tdrtargets.org](http://www.tdrtargets.org/)  
ChEMBLdb: [https://www.ebi.ac.uk/chembl/beta/](https://www.ebi.ac.uk/chembl/beta/)  
ChEMBL Blog: [http://chembl.blogspot.com/](http://chembl.blogspot.com/)  
  
  
[![](https://www.ebi.ac.uk/chemblntd/static/images/general/wtlogo.png)](http://www.wellcome.ac.uk/)       [![](https://www.ebi.ac.uk/chemblntd/static/images/general/mmv-logo.png)](http://www.mmv.org/) ![spacer](https://www.ebi.ac.uk/inc/images/spacer.gif)

